### PR TITLE
[3.2][ticket/12288] Add ability to create profile urls containing full board url

### PR DESCRIPTION
**Pull request by Elsensee**
http://tracker.phpbb.com/browse/PHPBB3-12288

### CODE REVIEW

File: tests/functions/obtain_online_test.php [hunk 1]
```diff
@@ -27,6 +27,11 @@ protected function setUp()
 		$db = $this->db = $this->new_dbal();
 		$config = array(
 			'load_online_time'	=> 5,
+			'force_server_vars'	=> 1,
```
Comment (marc1706) [inline tests/functions/obtain_online_test.php]: @nickvergessen @bantu I'm pretty sure this hasn't been answered?

**Comment by marc1706** [comment]
Needs rebase

**Comment by Nicofuma** [comment]
status? IIRC one question is still pending @nickvergessen @bantu @marc1706 @Elsensee 

Note: needs to be rebased onto master

**Comment by Elsensee** [comment]
@Nicofuma I am not sure if I remember it correctly but I think this function should be splitted into several functions inside a new/existing class.

**Comment by Nicofuma** [comment]
which one?

**Comment by Elsensee** [comment]
I think we could put it into user_loader class which currently has a wrapper function for get_username_string();
That way we can even get rid of these ugly static variables in the function. :)

**Comment by Nicofuma** [comment]
I see

**Comment by Nicofuma** [comment]
anyway you need to rebase on master and send a new PR :)

**Comment by marc1706** [comment]
IMHO this is a rather minor change that could be added after feature freeze.

**Pull request closed without merging**